saleor/core/sitemaps.py

```diff
@@ -1,30 +1,36 @@
+from django.conf import settings
 from django.contrib.sitemaps import Sitemap
 
 from ..page.models import Page
 from ..product.models import Category, Collection, Product
 
 
-class ProductSitemap(Sitemap):
+class i18nSitemap(Sitemap):
```

Comment:
This would look better and more pythonic as either `I18nSitemap` or `I18NSitemap`.